java_client/ChitChatDesktop/src/com/moc/chitchat/view/BaseView.java

```diff
@@ -43,43 +44,78 @@ public Scene getScene() {
         base.add(this.buildHeader(), "dock north");
 
         MigPane contentPane = this.getContentPane();
+        contentPane.setId("base-content-pane");
         base.add(contentPane, "grow");
 
         base.add(this.buildFooter(), "dock south");
-        return new Scene(base, this.width, this.height);
+
+        Scene scene = new Scene(base, this.width, this.height);
+        scene.getStylesheets().add("https://fonts.googleapis.com/css?family=Lato");
+        scene.getStylesheets().add("https://fonts.googleapis.com/css?family=Fredericka+the+Great");
+        scene.getStylesheets().add("https://fonts.googleapis.com/css?family=Playfair+Display:700");
+        scene.getStylesheets().add("https://fonts.googleapis.com/css?family=Pacifico");
+        scene.getStylesheets().add(this.getClass()
+                .getResource("/com/moc/chitchat/resources/chitchat.css")
+                .toExternalForm());
+
+        return scene;
     }
 
     private MigPane buildHeader() {
         MigPane header = new MigPane();
         header.setLayout("fill");
-        header.setStyle("-fx-background-color: #32292F");
-
+        header.setStyle("-fx-background-color:  #3C4F76;");
+        header.setId("base-header");
         Label title = new Label("Chit Chat");
-        title.setTextFill(Color.WHITE);
+        title.setWrapText(true);
+        title.setId("base-header-title");
+        if (this.baseStage.getConfiguration() != null
+                && this.baseStage.getConfiguration().getLoggedInUser() != null) {
+
+            logout = new Button("Log out");
+            logout.setOnAction(this);
+            logout.setStyle("-fx-background-color: transparent");
+            logout.setId("logoutbtn");
```

Comment:
Should follow the existing form (*view*-*name*-*control type*) to be `base-logout-btn`.